server/segment_rpc.go

```diff
@@ -386,9 +384,19 @@ func SubmitSegment(sess *BroadcastSession, seg *stream.HLSSegment, nonce uint64)
 		return nil, err
 	}
 
-	// TODO(yondonfu):
-	// Set balUpdate.Debit = fee computed based on returned results
-	// Set balUpdate.Status = ReceivedChange
+	// We treat a response as "receiving change" where the change is the difference between the credit and debit for the update
+	balUpdate.Status = ReceivedChange
+	priceInfo := sess.OrchestratorInfo.PriceInfo
+	if priceInfo != nil {
```

Comment:
`priceInfo` would be `nil` when the node is running in off-chain mode because Os don't include a price in the `OrchestratorInfo` message in off-chain mode

Reply:
Right, I tend to always forget about offchain mode existing😅 . Thanks !